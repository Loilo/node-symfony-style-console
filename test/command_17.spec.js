

const wrapper = require('./wrapper')

test('17 Ensure symfony style helper methods handle trailing backslashes properly when decorating user texts', wrapper(io => {
	io.title('Title ending with \\')
	io.section('Section ending with \\')
}))
