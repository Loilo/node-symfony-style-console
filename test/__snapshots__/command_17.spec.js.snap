// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`17 Ensure symfony style helper methods handle trailing backslashes properly when decorating user texts 1`] = `
"
Title ending with \\\\
===================

Section ending with \\\\
---------------------

"
`;
